src/cloud-priv.yml

```diff
@@ -62,6 +76,34 @@ components:
       $ref: './common/schemas/Links.yml'
     OrganizationRequest:
       $ref: './cloud/schemas/OrganizationRequest.yml'
+    BucketLimits:
+      $ref: "./quartz/schemas/BucketLimits.yml"
+    CheckLimits:
+      $ref: "./quartz/schemas/CheckLimits.yml"
+    DashboardLimits:
+      $ref: "./quartz/schemas/DashboardLimits.yml"
+    NotificationEndpointLimits:
+      $ref: "./quartz/schemas/NotificationEndpointLimits.yml"
+    NotificationRuleLimits:
+      $ref: "./quartz/schemas/NotificationRuleLimits.yml"
+    OrganizationWithToken:
+      $ref: "./quartz/schemas/OrganizationWithToken.yml"
+    Organization:
+      $ref: "./quartz/schemas/Organization.yml"
+    Organizations:
+      $ref: "./quartz/schemas/Organizations.yml"
+    OrgLimits:
+      $ref: "./quartz/schemas/OrgLimits.yml"
+    RateLimits:
+      $ref: "./quartz/schemas/RateLimits.yml"
+    RestrictedLimit:
+      $ref: "./quartz/schemas/RestrictedLimit.yml"
+    TaskLimits:
+      $ref: "./quartz/schemas/TaskLimits.yml"
+    Unlimited:
+      $ref: "./quartz/schemas/Unlimited.yml"
+    QuartzLimit:
+      $ref: "./quartz/schemas/Limit.yml"
```

Comment:
I just realized that these are pointing at the Quartz limit structs. I'm not sure that's correct here. I think this should be IDPE limit structs. They should be roughly the same, and the intention here is that Quartz should do any translation necessary between the two structs. In the past, there hasn't been any translation necessary.

Reply:
I don't understand why we would have different definitions. This is an object that is used to communicate between quartz and idpe. What other APIs are using these limits?

Reply:
check it now

Reply:
Looks good from what I can tell. Let us know when the corresponding change to IDPE goes in (the one that doesn't require us to supply an IDPE ID for the Limit struct).